### ISSUE TITLE
Inherit webcovers up to ancestors

### ISSUE THREAD

**Issue by brendanheywood** [theCrag/website#3062]
**What happened?**

We want webcovers at many levels, eg a webcover for australia. For people visiting australia a webcover of the most iconic crag in aus, eg araps, or the totem pole, might make sense. So if a webcover is missing we could automatically find the most iconic crags and inherit their webcovers. And we could also grab 2-3 webcovers and rotate them. Something like the lonely-planet fadeout:

https://www.lonelyplanet.com/greece

**What you expected:**

* [ ] regions with no webcovers have webcovers automatically selected for them
* [ ] selection is based of a combination of both kudos of the crag, and ideally the quality of the webcover (which we don't have at the moment)

**Comment by scd** [comment]
If the area already has a list of iconic crags then and easy solution would be to make sure the webcovers are included in the iconic crags. This will be very simple to implement and I don't have to worry about writing any webcover rollup code. The drawback is that some regions may not have web covers if all their descendant webcovers are not in one of the iconic crags. @brendanheywood are you ok with this, at least as an intermediate solution?

**Comment by brendanheywood** [comment]
Yeah I was actually expecting that as the quickest interim implementation. It won't have the coverage of a deeper method but will be pretty good.

The one deficiency is that if we are only loading this in the iconic crags data, that means this won't be available in other contents, for instance if we were looking at a search facet page for a region then it won't show the webcover there? Maybe that it actually a feature as you could then up with many nodes in the facet page showing the same webcover so their value is reduced in that context.

The other things that I like about the iconic crag concept is that in the rotating banner we could actually show the crag name and if you click on the webcover you go to that crag and not to the webcover photo page.

**Comment by lordyavin** [comment]
+1 for the lonely-planet fadeout
